lib/util/json.js

```diff
@@ -32,5 +32,32 @@ function readJsonAsync(file) {
         });
 }
 
+function convertJsonToList(orgJson, level){
```

Comment:
how about JsonToList?
Even without "convert", it seems to have enough meaning.
if you agree this, I hope change all of convertJsonToList to JsonToList.

Reply:
@ssuminahn , Good opinion. 
I thought "JsonToList" such as "atoi", too.
The reason is to add "convert"  to "JsonToList" is that other function name are  "readJsonSync, readJsonAsync".
So, I named it like this.

Please consider the reason. Let's have a talk.

Reply:
keep current.